### ISSUE TITLE
WCAG 2.2 Success Criteria consideration: Require disabled icon for disabled UI interface elements

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag#661]
I know one thing that doesn't work well for people with low vision is disabled interface elements. We've exempted them from 1.4.3 contrast, but maybe a better way is to provide an icon for a disabled UI element. 

So it would have to have at least 3:1 contrast OR add an icon to indicate it is disabled,

**Comment by bkimmel** [comment]
I'd second this: Especially when disabled/inactive elements are used to communicate the "potential" of a certain interaction (e.g. after you fill out these things you can "submit" this). This is a common UX pattern and it creates a _huge_ safe harbor for things that cut against the grain of the "Perceivable" goal. I'm not certain that an "icon" _per se_ is the right way to address it, but I think it's a problem that deserves some serious consideration.

**Comment by alastc** [comment]
Taking this off the 2.2 slate, needs more work/consideration/research.

**Comment by patrickhlauke** [comment]
FWIW i'd be far more inclined to say some form of minimum contrast should apply (to solve the readability aspect), but not sold on the "icon" idea. we can't start demanding icons everywhere. some more general intent of "indicating that something is disabled" may be more flexible and doesn't start impinging far too much  back on WCAG demanding certain visual designs. also noting that there may be a cross-over here with how user agents themselves indicate things again...
